apps/web/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "web",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@fluentui-react-native/checkbox": "0.3.50",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "@types/jest": "^24.0.0",
+    "@types/node": "^12.0.0",
+    "@types/react": "^16.9.0",
+    "@types/react-dom": "^16.9.0",
+    "react": "16.11.0",
+    "react-art": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-native": "0.62.2",
+    "react-native-web": "^0.12.3",
+    "typescript": "3.8.3",
+    "webpack": "4.42.0"
+  },
+  "scripts": {
+    "web": "webpack-dev-server"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+  "browserslist": {
```

Comment:
Can you remind me what these production and development sections are for?

Reply:
This was part of the create-react-app initial configuration but it seems to work fine without these production and development sections now, so I'll take them out.